submissions/DmytryjK/task_js_oop/js/app.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+const baseConfig = {
+    fieldWidth: 400,
+    fieldHeight: 390, 
+    heroWidth: 100,
+    heroHeight: 170,
+    enemyWidth: 100,
```

Comment:
I feel there is a connection between some of this numbers.
As a lazy peer developer I'd like to make changes to as few entities as only possible.

Reply:
Fixed)